test/storage.test.ts

```diff
@@ -249,4 +249,43 @@ describe("Regression", () => {
       await storage.clear();
     }
   });
+
+  it("prefixStorage getItems returns null (issue #396)", async () => {
```

Comment:
is this test description wrong/old? 👀 

since it seems we're testing that `getItems` returns values and non-prefixed keys

Reply:
It's old. I used this test to replicate the bug